src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveReviewsResourceTest.kt

```diff
@@ -134,7 +134,7 @@ class IncentiveReviewsResourceTest : IncentiveLevelResourceTestBase() {
         .uri("/incentives-reviews/prison/MDI/location/MDI-1/level/STD?sort=PRISON")
         .headers(setAuthorisation(roles = listOf("ROLE_INCENTIVES")))
         .exchange()
-        .expectErrorResponse(HttpStatus.BAD_REQUEST, "No enum constant")
+        .expectErrorResponse(HttpStatus.BAD_REQUEST, "Parameter conversion failure: 400 BAD_REQUEST \"Type mismatch.\"")
```

Comment:
if "No enum constant" doesn't appear _anywhere_ including the "cause", then this seems reasonable. though i'd only search for "Type mismatch".

related to https://github.com/ministryofjustice/hmpps-incentives-api/pull/407/files#r1159704464